test/agents.test.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const path = require('path');
+const { Validator } = require('jsonschema');
+
+const schemaPath = path.resolve(__dirname, '..', 'test-resources', 'agents-schema.json');
+const agentsPath = path.resolve(__dirname, '..', 'protocol', 'agents.json');
+const schema = require(schemaPath);
+const agents = require(agentsPath);
+
+const jsonschema = new Validator();
+jsonschema.addSchema(
+    schema,
+    'https://schemas.ably.com/json/asset-common/Agents'
+);
+
+describe("agents.json", () => {
```

Comment:
I recognise I am being pedantic, but in this file we're mixing single quotes and double quotes for strings.

Reply:
Ah, my bad. I'll make sure to get prettier running on JS files when we resolve #79 which will fix this along with any other styling inconsistencies.